e2e-tests/support/trade-finance-manager/overrideRedisUserSession.js

```diff
@@ -0,0 +1,3 @@
+module.exports = (sessionIdentifier, value, maxAge) => {
+  cy.task('overrideRedisUserSession', { sessionIdentifier, value, maxAge });
+};
```

Comment:
Should this be `overrideTfmRedisUserSession`, similar to below (`overrideTfmUserSessionId`)?